### PR TITLE
fixed broken_link of gatsby starter #21720

### PR DESCRIPTION
**Pull request by Usamaliaquat123**
issue #21720

### CODE REVIEW

**Comment by Usamaliaquat123** [comment]
@tesseralis

File: docs/docs/gatsby-cli.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ gatsby new [<site-name> [<starter-url>]]
 | Argument    | Description                                                                                                                                                                                                     |
 | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | site-name   | Your Gatsby site name, which is also used to create a project directory.                                                                                                                                        |
-| starter-url | A Gatsby starter URL or local file path. Defaults to [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); see the [Gatsby starters](/docs/gatsby-starters/) docs for more information. |
+| starter-url | A Gatsby starter URL or local file path. Defaults to [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); see the [Gatsby starters](/docs/docs/starters.md) docs for more information. |
```
Comment (tesseralis) [inline docs/docs/gatsby-cli.md]: If you go to https://gatsbyjs.org/docs/docs/starters.md you'll see that the page doesn't exist. The correct URL is `/docs/starters/`. Could you update it to the correct URL?

```suggestion
| starter-url | A Gatsby starter URL or local file path. Defaults to [gatsby-starter-default](https://github.com/gatsbyjs/gatsby-starter-default); see the [Gatsby starters](/docs/starters/) docs for more information. |
```

Comment (Usamaliaquat123) [reply docs/docs/gatsby-cli.md]: #21724

**Comment by Usamaliaquat123** [comment]
#21738

**Comment by Usamaliaquat123** [comment]
#tesseralis   sorry about that #21738 this must be a another PR ;)

**Comment by laurieontech** [comment]
Thanks for the PR! While I believe the starters link correction is accurate, I'm less confident in the change to the footnote. As mentioned in the original issue this is a departure from other docs and likely should be refactored entirely as the links are still challenging to reference.

Given that these pages are unrelated, I'd recommend splitting this PR into two so we can address the starters link fix and figure out a better option for the footnote.

**Comment by Usamaliaquat123** [comment]
> Thanks for the PR! While I believe the starters link correction is accurate, I'm less confident in the change to the footnote. As mentioned in the original issue this is a departure from other docs and likely should be refactored entirely as the links are still challenging to reference.
> 
> Given that these pages are unrelated, I'd recommend splitting this PR into two so we can address the starters link fix and figure out a better option for the footnote.

sure

**Comment by laurieontech** [comment]
Great, let's merge this one.

**Comment by LorenAmelang** [comment]
In case anyone is wondering... The mention in "npm config set python python2.7" ??? #21920 was accidental. The "similar" bot popped up a link to this issue, and somehow it got embedded in my new issue. I edited it back out, but this apparently stuck. Sorry!

**Pull request merged**